docs/pr-previews.md

```diff
@@ -0,0 +1,32 @@
+# Pull Request Previews
+
+Pull requests to the React SDK layer (and in the future other layers as well)
+automatically set up a preview site with a full deployment of Element with the
+changes from the pull request added in so that anyone can easily test and review
+them. This is especially useful for checking visual and interactive changes.
+
+To access the preview site, scroll down to the bottom of the PR where the
+various CI results are displayed:
+
+![Pull request: checks section](./img/pr-checks.png)
+
+The checks section could be collapsed at first, so you may need to click "Show
+all checks" to reveal them. Look for an entry that mentions `deploy-preview`. It
+may be at the end of the list, so you may need scroll a bit to see it. To access
+the preview site, click the "Details" link in the deploy preview row.
+
+**Important:** Please always use test accounts when logging into preview sites,
+as they may contain bugs or other issues that could damage your account.
```

Comment:
it might be good to raise this warning up to "unreviewed and potentially dangerous code" level.

Reply:
Okay, works for me. I wasn't sure how loud to shout, but seems like it's agreed that lots of shouting is warranted. 😄